src/pages/Header/Header.jsx

```diff
@@ -53,21 +56,20 @@ function Header({ setTerm }) {
       <Menu.Menu position="right">
         <Item>
           <Icon name="sun" />
-          <Checkbox slider />
+          <Checkbox slider onChange={() => setAppContext({...appContext, themeIsDark: !appContext.themeIsDark})} />
           <Icon name="moon" />
         </Item>
         {!authenticated ? (
           <Item
             name="log in"
-            onClick={() => {
-              history.push('/login');
-            }}
+            onClick={() => setOpen(true)}
           />
         ) : (
-          <UserMenu item text={user ? user.username : '@wizeline'}>
+          <UserMenu item text={user ? user.name : '@username'}>
             <Dropdown.Menu>
-              <Dropdown.Item>Home</Dropdown.Item>
-              <Dropdown.Item>Favorites</Dropdown.Item>
+              <Dropdown.Item onClick={() => history.push('/home')}>Home</Dropdown.Item>
+              <Dropdown.Item onClick={() => history.push('/favorites')}>Favorites</Dropdown.Item>
+              <Dropdown.Item onClick={() => history.push('/watchLater')}>Watch later</Dropdown.Item>
```

Comment:
this part is better defined with a loop, it will be easier to maintain than defining each item manually, for example:

```jsx
const items = [
  { route: '/home', label: 'Home' },
  { route: '/favorites', label: 'Favorites' },
  { route: '/watchLater', label: 'Watch Later' },
];

...

{items.map(({route, label}) => (
  <Dropdown.Item onClick={() => history.push(route)}>{label}</Dropdown.Item>
)}

...
```